src/main/java/taxi/controller/car/AddDriverToCarController.java

```diff
@@ -30,6 +30,6 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         Driver driver = driverService.get(driverId);
         Car car = carService.get(carId);
         carService.addDriverToCar(driver, car);
-        resp.sendRedirect(req.getContextPath() + "/cars/drivers/add");
+        resp.sendRedirect(req.getContextPath() + "/cars");
```

Comment:
looks like you have similar end point for AddCarController and AddDriverToCarController

Reply:
<img width="1231" alt="Screenshot 2022-11-25 at 16 37 53" src="https://user-images.githubusercontent.com/92673432/204007180-65db5910-2d56-41b0-ba07-ea541bfc3567.png">

Reply:
yeah, sorry my bad, was not attentive